### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by Nuga0909**
- Create project section with DOM manipulation
- create popups with DOM manipulation
- make buttons functional
- update CSS styling

### CODE REVIEW

**Review by sumairq — COMMENTED** [review]
### Hi @Nuga0909  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, in which there are conflicts in the pull request. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: home.js [hunk 1]
```diff
@@ -10,4 +10,195 @@ window.onload = function () {
...
+  sourceButton.innerText = 'See Source';
+  sourceButton.className = 'btn-pop';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
+    sourceButton,
```
Comment (thecodechaser) [inline home.js]: - [x] Please fix the below issues for the mobile screens and make sure your model(pop-up) design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DB34JCuHQzSTpBg8-0) design.
  - Please make sure the buttons in the pop-up are aligned in a single row.
  - Please add the icons in the buttons as in the Figma design.
  - Please use the correct color for the buttons. `#FF6B00`
  - Please apply the hover effect on the buttons.
    - image



### Your design
image

### Figma design
image

File: home.js [hunk 2]
```diff
@@ -10,4 +10,195 @@ window.onload = function () {
...
+  const sourceButton = document.createElement('button');
+  sourceButton.innerText = 'See Source';
+  sourceButton.className = 'btn-pop';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
```
Comment (thecodechaser) [inline home.js]: - [x] For the desktop screens, The model(pop-up) doesn't match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DB34JCuHQzSTpBg8-0) design and I can't see the project details. Please make sure that the project details are visible on the desktop screens and design matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=DB34JCuHQzSTpBg8-0) template.


### Your design
image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Nuga0909, and @bernabasy👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: home.js (hunk 1 above)
Comment (Nuga0909) [reply home.js]: I  had to change the color due to accessibility issues.

File: home.js [hunk 3]
```diff
@@ -10,4 +10,195 @@ window.onload = function () {
...
+  sourceButton.className = 'btn-pop2 hover';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
+    sourceButton,
+  );
```
Comment (ha-manel) [inline home.js:125]: As requested by the previous reviewer:
- [ ] kindly use the correct `background-color` for the buttons (`#FF6B00`)
- [ ] kindly add the icons to the buttons as shown in Figma

[ Optional ]

- [ ] Good job on making the buttons in a single row, however one of them is lower than the other, please try to make them aligned on the same level 👍 

## Your design:
image

## Figma design:
image

Comment (ha-manel) [inline home.js:125]: - [ ] In desktop version, the content is overflowing the popup and we still can't see the content (the buttons) like the previous reviewer mentioned, kindly fix the issue 👍 

[ Optional ]
- [ ] It's better if we can't scroll in the website when the popup window is open, that would make the experience nicer and smoother 💯 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Nuga0909 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 

## Required Changes:
_Kindly check the comments below_

## Optional:
- [ ] The previous reviewer didn't notice that you haven't setup `Eslint` for this project so I'm putting this in optional but please add it to your linters setup as it's part of the requirements 👍  

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — COMMENTED** [review]
Hello @Nuga0909 & @bernabasy 👋,

_This is an additional review as a result of the second opinion request._

- Nice implementation of the pop-up modal 👍 

   - As part of the project requirement, you should stick to the design as much as possible including font, colours, images, text, and margins.

> Screenshot 2022-11-18 at 18 40 18

- Presently, this is how the pop-up looks without chrome inspector.
  >  Screenshot 2022-11-18 at 18 50 15


- I strongly suggest you implement the previously requested changes 📶 

Keep rocking 🚀!

**Review by MahmoudBakr23 — APPROVED** [review]
Hi team 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done working on the requested changes 
- Great communications 

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**